commit 9b37074f7609cd496c067e88ef8c436981aa7267
Author: mjw <mjw@a5019735-40e9-0310-863c-91ae7b9d1cf9>
Date:   Thu Jun 29 15:26:30 2017 +0000

    memcheck/tests: Use ucontext_t instead of struct ucontext
    
    glibc 2.26 does not expose struct ucontext anymore.
    
    Signed-off-by: Khem Raj <raj.khem@gmail.com>
    
    git-svn-id: svn://svn.valgrind.org/valgrind/trunk@16457 a5019735-40e9-0310-863c-91ae7b9d1cf9

diff --git a/memcheck/tests/linux/stack_changes.c b/memcheck/tests/linux/stack_changes.c
index a978fc2..7f97b90 100644
--- a/memcheck/tests/linux/stack_changes.c
+++ b/memcheck/tests/linux/stack_changes.c
@@ -10,7 +10,7 @@
 // This test is checking the libc context calls (setcontext, etc.) and
 // checks that Valgrind notices their stack changes properly.
 
-typedef  struct ucontext  mycontext;
+typedef  ucontext_t  mycontext;
 
 mycontext ctx1, ctx2, oldc;
 int count;
